doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -685,7 +1385,7 @@ Note that the sfcf* and atmf* files are not output on the 6 tiles, but instead a
 
 The regional configuration will generate similar output files, but the *tile[1-6]* is not included in the filename.
 
-Two files (*model_configure* and *diag_table*) control the output that is generated by the UFS Weather Model.  The output files that contain the model variables are written to a file as shown in the figure below.  The format of these output files is selected in *model_configure* as NetCDF or nemsio.  The information in these files may be remapped, augmented with derived variables, and converted to GRIB2 by the Unified Post Processor (UPP).  Model variables are listed in the *diag_table* in two groupings, *fv3_history* and *fv3_history2d*, as described in :numref:`Section %s <diag_tableFile>`.  The names of the files that contain these model variables are specified in the *model_configure* file. When *quilting* is set to *.true.* for the write component, the variables listed in the groups *fv3_history* and *fv3_history2d* are converted into the two output files named in the *model_configure* file, e.g. *atmfHHH.* and *sfcfHHH.*. The bases of the file names (*atm* and *sfc*) are specified in the *model_configure* file, and *HHH* refers to the forecast hour.
+Two files (*model_configure* and *diag_table*) control the output that is generated by the UFS Weather Model.  The output files that contain the model variables are written to a file as shown in the figure below.  The format of these output files is selected in *model_configure* as NetCDF.  The information in these files may be remapped, augmented with derived variables, and converted to GRIB2 by the Unified Post Processor (UPP).  Model variables are listed in the *diag_table* in two groupings, *fv3_history* and *fv3_history2d*, as described in :numref:`Section %s <diag_tableFile>`.  The names of the files that contain these model variables are specified in the *model_configure* file. When *quilting* is set to *.true.* for the write component, the variables listed in the groups *fv3_history* and *fv3_history2d* are converted into the two output files named in the *model_configure* file, e.g. *atmfHHH.* and *sfcfHHH.*. The bases of the file names (*atm* and *sfc*) are specified in the *model_configure* file, and *HHH* refers to the forecast hour.
 
 .. image:: _static/fv3IO.png
```

Comment:
Can we put in a description/explanation for this figure?

Reply:
I can add a figure caption; I made a guess at a reasonable one. 

Also, I noticed the image file needs to be updated because we've renamed the output to atmf and sfcf instead of dynf and phyf.